employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java

```diff
@@ -0,0 +1,22 @@
+package com.artimes.employee.controller;
```

Comment:
Please add the Junit Tests for this class.

Reply:
I dont have any functionality to write Junit test cases